cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js

```diff
@@ -31,24 +35,14 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
 
   var BASE_CLASS = standardType.BEHAVIOR_PREFIX + 'flyout-menu';
   var SEL_PREFIX = '[' + standardType.JS_HOOK + '=' + BASE_CLASS;
-
   var BASE_SEL = SEL_PREFIX + ']';
-  var ALT_TRIGGER_SEL = SEL_PREFIX + '_alt-trigger]';
-  var CONTENT_SEL = SEL_PREFIX + '_content]';
-  var TRIGGER_SEL = SEL_PREFIX + '_trigger]';
-
-  // TODO: Update atomic-helpers to support CSS selectors for validity check.
-  var _dom = dataHook.contains( element, BASE_CLASS ) ? element : null;
-  if ( !_dom ) { _dom = element.parentNode.querySelector( BASE_SEL ); }
-  if ( !_dom ) { throw new Error( 'Selector not found on passed node!' ); }
-
-  var _triggerDom = _dom.querySelector( TRIGGER_SEL );
-  var _contentDom = _dom.querySelector( CONTENT_SEL );
 
-  if ( !_triggerDom ) { throw new Error( TRIGGER_SEL + ' is missing!' ); }
-  if ( !_contentDom ) { throw new Error( CONTENT_SEL + ' is missing!' ); }
+  // Verify that the expected dom attributes are present.
+  var _dom = behavior.checkBehaviorDom( element, BASE_CLASS );
+  var _triggerDom = behavior.checkBehaviorDom( element, BASE_CLASS + '_trigger' );
+  var _contentDom = behavior.checkBehaviorDom( element, BASE_CLASS + '_content' );
```

Comment:
Mm, that's a good question. I'd say pressing the trigger performs the action of showing and hiding the content, and therefore the contentDom is the content, not the action.